src/components/cards/BaseStakingCard.tsx

```diff
@@ -117,6 +127,25 @@ const BaseStakingCard: FC<IBaseStakingCardProps> = ({
 							`GIVgarden `}
 						{type}
 					</StakingPoolExchange>
+					{walletNetwork === config.XDAI_NETWORK_NUMBER &&
+					type === StakingType.GIV_LM ? (
+						<IconWithTooltip
+							direction={'top'}
+							icon={
+								<IconHelp
+									color={brandColors.deep[100]}
+									size={12}
+								/>
+							}
+						>
+							<GIVgardenTooltip>
+								While staking GIV in this pool you are also
+								granted voting power (gGIV) in the GIVgarden.
+							</GIVgardenTooltip>
+						</IconWithTooltip>
+					) : (
+						' '
+					)}
```

Comment:
@divine-comedian 
Please use this pattern:
{condition 1 && condition2 && <Component />}